hack/verify-api-conversion.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+cd "${REPO_ROOT}" || exit 1
```

Comment:
I wonder if we should instead introduce a scripts/ci-verify.sh which calls `make verify`

`make verify` would then additionally also call `$(make) verify-conversions)`

(Mostly to align with other providers and to have a single verify job instead of one per validation)

Reply:
Yes, I just saw your other comment on the test-infra PR too! 
I think it makes sense to unify all the `verify` make targets :) Thanks @sbueringer!

Reply:
Just to mention it here to. Had a short discussion yesterday about the jobs in the CAPV office hours and we would keep the separate jobs. Goal is to make it easier to see on PRs which verification fails.

Additionally we want to have a verify target which can be used for convenience locally to run all verifications at once.